app/mailers/tasks_mailer.rb

```diff
@@ -0,0 +1,15 @@
+class TasksMailer < ApplicationMailer
+  default template_path: 'mailers/tasks'
+
+  def task_completed(task, current_user)
+    @task = task
+    @user = current_user
+    watchers = @task.watchers.map {|watcher| watcher.email if watcher != current_user}
```

Comment:
this code is incorrect
you can have [nil] when user = current_user in array

use

watchers = @task.watchers.where.not(users: {id: @user.id}}).pluck(:email)

I think it should work